### PR TITLE
Add PHOBOSSRC urls to std.experimental

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by wilzbach** [comment]
https://github.com/dlang/phobos/pull/6007 is needed first

**Comment by JackStouffer** [comment]
Why should this go on stable?

**Comment by wilzbach** [comment]
Otherwise it will take two months before it's visible on dlang.org

**Comment by JackStouffer** [comment]
But it makes the merge back into master more complex. I'm skeptical of putting anything bug regression fixes onto stable. CC @MartinNowak

**Comment by wilzbach** [comment]
It doesn't. The other way - cherry-picking from master - makes it a bit more complicated. I can submit a merge back into master easily afterwards to show that we don't need to be afraid about this.

**Comment by wilzbach** [comment]
>  I can submit a merge back into master easily afterwards

See also: https://github.com/wilzbach/git-tools/blob/master/merge_stable

**Comment by JackStouffer** [comment]
I'm assuming this macro is not used for the ddox version because of the auto generated link at the top right?

**Comment by wilzbach** [comment]
> I'm assuming this macro is not used for the ddox version because of the auto generated link at the top right?

Yeah:

http://dtest.dlang.io/artifact/website-110116a6991ae150e89b6abd767a492de920c581-a89969b4d0a4227eed98a2eb0190bd10/web/library-prerelease/std/experimental/allocator/typed.html
http://dtest.dlang.io/artifact/website-110116a6991ae150e89b6abd767a492de920c581-a89969b4d0a4227eed98a2eb0190bd10/web/phobos-prerelease/std_experimental_allocator_typed.html

Bit it still really sucks that we have to do such trivial things manually.

File: std/experimental/allocator/building_blocks/ascending_page_allocator.d [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Written in the D programming language.
+/**
+Source: $(PHOBOSSRC std/experimental/allocator/building_blocks/_ascending_page_allocator.d)
+*/
```
Comment (wilzbach) [inline std/experimental/allocator/building_blocks/ascending_page_allocator.d]: Meh this is ugly:

image

image


http://dtest.dlang.io/artifact/website-110116a6991ae150e89b6abd767a492de920c581-a89969b4d0a4227eed98a2eb0190bd10/web/phobos-prerelease/std_experimental_allocator_building_blocks_ascending_page_allocator.html

I guess I will remove exposing the module via Ddoc from this PR.

CC @Darredevil

Comment (wilzbach) [reply std/experimental/allocator/building_blocks/ascending_page_allocator.d]: -> https://github.com/dlang/phobos/pull/6009

**Comment by wilzbach** [comment]
> I can submit a merge back into master easily afterwards
> See also: https://github.com/wilzbach/git-tools/blob/master/merge_stable

https://github.com/dlang/phobos/pull/6006

**Pull request merged**